xbmc/network/NetworkServices.cpp

```diff
@@ -624,6 +648,11 @@ bool CNetworkServices::StartServer(enum ESERVERS server, bool start)
       ret = settings->SetBool(CSettings::SETTING_SERVICES_ZEROCONF, start);
       break;
 
+    case ES_WSDISCOVERY:
+      // the callback will take care of starting/stopping zeroconf
```

Comment:
Typo from copy/paste, you likely mean wsdiscovery on the comment and not zeroconf